### PR TITLE
Download button

### PR DESCRIPTION
**Pull request by jespicas**
DownloadButton: Add Cypress Test 
Check if exist attribute download with resume.json
Check if href contains a value
Screenshot from 2020-10-07 22-20-48

### CODE REVIEW

File: cypress/integration/components/Tools/Buttons/DownloadButton.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...evel%22%3A%204%0A%09%09%09%09%7D%0A%09%09%09%5D%2C%0A%09%09%09%22isVisible%22%3A%20true%2C%0A%09%09%09%22columnWidthPercent%22%3A%20%2233.33%25%22%0A%09%09%7D%0A%09%5D%2C%0A%09%22projects%22%3A%20%5B%0A%09%09%7B%0A%09%09%09%22name%22%3A%20%22Project%201%22%2C%0A%09%09%09%22dateFrom%22%3A%20%22XX%2FXXXX%22%2C%0A%09%09%09%22dateTo%22%3A%20%22%22%2C%0A%09%09%09%22link%22%3A%20%22http%3A%2F%2Fwebsite.com%22%2C%0A%09%09%09%22teamBrief%22%3A%20%221-person%20project%22%2C%0A%09%09%09%22details%22%3A%20%5B%0A%09%09%09%09%22Detail%201%22%2C%0A%09%09%09%09%22Detail%202%22%2C%0A%09%09%09%09%22http%3A%2F%2FprojectLink.com%22%0A%09%09%09%5D%2C%0A%09%09%09%22isVisible%22%3A%20true%0A%09%09%7D%2C%0A%09%09%7B%0A%09%09%09%22name%22%3A%20%22Project%202%22%2C%0A%09%09%09%22link%22%3A%20%22http%3A%2F%2Fwebsite.com%22%2C%0A%09%09%09%22dateFrom%22%3A%20%22XX%2FXXXX%22%2C%0A%09%09%09%22dateTo%22%3A%20%22XX%2FXXXX%22%2C%0A%09%09%09%22teamBrief%22%3A%20%221-person%20project%22%2C%0A%09%09%09%22details%22%3A%20%5B%0A%09%09%09%09%22Detail%201%22%2C%0A%09%09%09%09%22Detail%202%22%2C%0A%09%09%09%09%22https%3A%2F%2Fgoogle.com%22%0A%09%09%09%5D%2C%0A%09%09%09%22isVisible%22%3A%20true%0A%09%09%7D%0A%09%5D%0A%7D')
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/DownloadButton.spec.js:12]: Please put the file data into a fixture.

File: cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress" />
+
+
+context('Click SaveToCloud buttons', () => {
```
Comment (aamay001) [inline cypress/integration/components/Tools/Buttons/SaveToCloudButtons.spec.js:4]: This file should not be included in this PR.

**Review by aamay001 — CHANGES_REQUESTED** [review]
Please make the changes specified below.

**Review by aamay001 — CHANGES_REQUESTED** [review]
Tests are failing because the `SaveToCloudButtons.spec` file is still included in this PR. Those tests fail. Please remove the file from the PR.

**Comment by jespicas** [comment]
> Tests are failing because the `SaveToCloudButtons.spec` file is still included in this PR. Those tests fail. Please remove the file from the PR.

Now not has this file and still failing

**Review by aamay001 — APPROVED** [review]
Thanks for making the changes. I can't figure out wh but, when the pipeline runs for your branch, it's not correctly populating a variable for Cypress. I ran the tests manually on your branch and they pass when I do that so I will merge your PR. Thanks for your contributions.

**Pull request merged**